crates/optimism/evm/src/lib.rs

```diff
@@ -187,7 +186,10 @@ where
             prevrandao: Some(attributes.prev_randao),
             gas_limit: attributes.gas_limit,
             // calculate basefee based on parent block's gas usage
-            basefee: next_block_base_fee(self.chain_spec(), parent, attributes.timestamp)?,
+            basefee: self
+                .chain_spec()
+                .next_block_base_fee(&parent, attributes.timestamp)
+                .unwrap_or_default(),
```

Comment:
I think this would be a bit problematic because now an error is treated as 0 value, imo we can keep using the regular next_block_base_fee function here